.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: "Close stale issues"
+on:
+  schedule:
+    # hourly
+    - cron: "0 * * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          only-labels: "Response Requested"
```

Comment:
Or alternatively `More Info Needed`? It's for situations where a community proposal has come in but it's possibly already implemented/belongs elsewhere/inadequately described.